.pre-commit-config.yaml

```diff
@@ -13,4 +13,5 @@ repos:
     hooks:
       - id: flake8
         exclude: "migration/.*|.git|viz-lib|node_modules|migrations|bin/upgrade"
```

Comment:
I think we should delete this line. And test without using it

Reply:
Yeah, I think you're right.  In theory, if it's truly parsing and using the options in setup.cfg then this line won't be needed.

Saying that because I thought we already tried the "args" option below, and it didn't work for some unknown reason.  (Note that I'm only about 50% confident we tried the same option as below, it might have been some other options we tried and I'm mis-remembering exactly what.)